pkg/destroy/bootstrap/bootstrap.go

```diff
@@ -43,21 +46,31 @@ func Destroy(dir string) (err error) {
 	}
 	defer os.RemoveAll(tempDir)
 
+	extraArgs := []string{}
 	for _, filename := range copyNames {
-		err = copy(filepath.Join(dir, filename), filepath.Join(tempDir, filename))
+		sourcePath := filepath.Join(dir, filename)
+		targetPath := filepath.Join(tempDir, filename)
+		err = copy(sourcePath, targetPath)
 		if err != nil {
+			if os.IsNotExist(err) && err.(*os.PathError).Path == sourcePath && filename == tfPlatformVarsFileName {
```

Comment:
Hmm, i think we need it for all the current platforms, currently right?

Reply:
> Hmm, i think we need it for all the current platforms, currently right?

`none` doesn't need it, although it's unlikely to end up in `destroy bootstrap`.  And per [this][1] (in flight, and for operators, but seems like a useful generic principle), we want to treat unrecognized platforms like `none` in most places.

[1]: https://github.com/openshift/installer/pull/1112/files#diff-6532666297c6115f7774f93ebab396c4R156

Reply:
can you include that in commit message or comment in code.

Reply:
> can you include that in commit message or comment in code.

Added commit-message discussion in 42a8a10 -> 6d449c5.